vector-config/src/main/res/values/config-settings.xml

```diff
@@ -42,6 +42,7 @@
     <bool name="settings_labs_thread_messages_default">false</bool>
     <bool name="settings_labs_new_app_layout_default">true</bool>
     <bool name="settings_labs_new_session_manager_default">false</bool>
+    <bool name="settings_labs_client_info_recording_default">false</bool>
```

Comment:
You could also create a `settings_labs_client_info_recording_visible` to toggle the visibility of this lab flag. Can be useful for other forks. See the doc at the top of the file.

Reply:
Okay, I see. I will update. I will add one also for the lab flag of new session manager as well.

Reply:
Fixed in [8613221](https://github.com/vector-im/element-android/pull/7352/commits/86132219d81f32f9329d2e1070455660a9878cc1)